sourced/ml/transformers/repo2weighted.py

```diff
@@ -1,16 +1,28 @@
+import operator
+
 from sourced.ml.transformers import Transformer
 
 
-class Repo2WeightedSet(Transformer):
-    def __init__(self, extractors, **kwargs):
+class Documents2BOW(Transformer):
```

Comment:
Why not weighted? It is weighted

Reply:
We discuss it when I was in Madrid. BOW = Weighted set. So just to avoid different namings for the same thing. BOW was the first.

Reply:
We urgently need a glossary of terms like I have in Apollo https://github.com/src-d/apollo/blob/master/doc/GLOSSARY.md